Gemfile

```diff
@@ -13,5 +13,6 @@ gem 'random-word'
 gem 'rspec'
 gem 'rubocop', '~> 0.73.0'
 gem 'rubocop-rspec'
+gem 'rubyXL'
```

Comment:
could you add a comment that this is for updating Excel spreadsheets?

Reply:
Done, just pushed up a comment in the gem file.